### ISSUE TITLE
2020 Openings!

### ISSUE THREAD

**Issue by ivanmarreta** [alagoasdev/eventos#1]
- Evento: Discussão 
- Local: Hangouts
- Data: à definir.
- Descrição: Discussão sobre assuntos que poderiam virar projectos ou meetups.
- Comunidade: AlagoasDev (Slack)

**Comment by endersonmaia** [comment]
além da **Data**, importante é o **Horário**, já que pelo visto, temos gente em fusos distintos

**Comment by ivanmarreta** [comment]
@endersonmaia Sim! Bem lembrado, mais importante até que a data! **Copiei e colei** o template lol

**Comment by pragmaticivan** [comment]
😢

**Comment by gmmcal** [comment]
Nem deixou fazer 1 ano pra reclamar! hahahahha
